constants.xml

```diff
@@ -239,6 +241,8 @@
         <OUTPUT_JSON>output.json</OUTPUT_JSON>
         <INPUT_CODE>input.scilla</INPUT_CODE>
         <ENABLE_SCILLA_MULTI_VERSION>true</ENABLE_SCILLA_MULTI_VERSION>
+        <FIELDS_MAP_DEPTH_INDICATOR>_fields_map_depth</FIELDS_MAP_DEPTH_INDICATOR>
+        <LOG_SC>true</LOG_SC>
```

Comment:
I think this should be set to false, otherwise we'll run into huge state problems again (due to printing it all).